hbase_sql/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactor.java

```diff
@@ -81,7 +83,8 @@
  *
  */
 public class SpliceDefaultCompactor extends DefaultCompactor {
-    private static final boolean allowSpark = true;
+    public static boolean allowSpark = true;
```

Comment:
Could you make this a construction parameter of SpliceDefaultCompactor instead? 
This non final `allowSpark` only for the sake of tests is a bit scary in my opinion.

Reply:
We can't make it a constructor parameter because the constructor is called directly by HBase, but I've made it a private non-static final field that gets initialized in the Constructor from the Configuration object.

Reply:
Neat!